queries/models/subsidio_onibus/schema.yml

```
@@ -0,0 +1,190 @@
+          where: "data between date('{date_range_start}') and date('{date_range_end}')"
+      - dbt_expectations.expect_table_aggregation_to_equal_other_table:
+          name: dbt_expectations.expect_table_aggregation_to_equal_other_table__sumario_faixa_servico_sentido_dia_pagamento
+          expression: count(distinct valor_penalidade)
+          compare_model: ref("valor_tipo_penalidade")
+          compare_expression: count(distinct valor)
+          group_by: [valor_penalidade]
+          compare_group_by: [-valor]
+          row_condition: "DATA BETWEEN DATE('{{ var('date_range_start') }}') AND DATE('{{ var('date_range_end') }}') AND valor_penalidade IS NOT null AND valor_penalidade != 0"
+          compare_row_condition: "valor IS NOT null AND valor != 0 and data_fim >= DATE('{{ var('date_range_start') }}') and data_inicio <= DATE('{{ var('date_range_end') }}')"
+          where: "1=1"
+          where: "data between date('{date_range_start}') and date('{date_range_end}')"
```

YAMLlint (1.37.1)

[error] duplication of key "where" in mapping

(key-duplicates)